commands/deploy.js

```diff
@@ -359,17 +360,15 @@ function logDetails(details, opts) {
       .value();
   }
 
-  if(details.runTestResult) {
+  if(details.runTestResult.any) {
```

Comment:
@kevinohara80 I need to revert this - I hadn't noticed runTestResult being returned in my debugging sessions, but it must be somewhere in the JSON because the original line still works.